src/workerd/api/r2-admin.h

```diff
@@ -38,7 +38,6 @@ class R2Admin: public jsg::Object {
           friend_tag_t)
```

Comment:
Looks like we can remove the `kj::String account` parameter on line 36: we aren't using it anymore.